otel-collector-config.yaml

```diff
@@ -16,8 +16,8 @@ exporters:
     endpoint: "http://zipkin-all-in-one:9411/api/v2/spans"
     format: proto
 
-  jaeger:
-    endpoint: jaeger-all-in-one:14250
+  otlp/jaeger:
+    endpoint: jaeger-all-in-one:4317
```

Comment:
why there is a change in port? I guess they have purposefully kept it 14250 to support nextJS framework.

Reply:
Frankly I don't know the nextJS framework.
New Jaeger versions support the OTLP protocol natively, and use that communication with the OTEL collector.

This port is not supposed to be seen or used by the observer application.

Reply:
Great fix. For further information: https://opentelemetry.io/blog/2023/jaeger-exporter-collector-migration/